packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -19,9 +19,16 @@ import { TelemetryHelper } from './telemetryHelper'
 import { maxRepoSizeBytes } from '../constants'
 import { isCodeFile } from '../../shared/filetypes'
 import { fs } from '../../shared'
+import { CodeWhispererSettings } from '../../codewhisperer'
 
 const getSha256 = (file: Buffer) => createHash('sha256').update(file).digest('base64')
 
+export async function checkForDevFile(root: string) {
+    const devFilePath = root + '/devfile.yaml'
```

Comment:
Do we need a path.join here? Will this work on windows?

Reply:
good catch, I can change it to use `path.join`

as for windows, I will get someone with a windows machine to test